code/modules/shuttle/marine_dropship.dm

```diff
@@ -832,711 +663,3 @@
-	icon_state = "blue_rearwing_llb"
-	opacity = FALSE
-
-/obj/structure/dropship_piece/two/rearwing/rightrbottom
-	icon_state = "blue_rearwing_rrb"
-	opacity = FALSE
-
-/obj/structure/dropship_piece/two/rearwing/leftllbottom
-	icon_state = "blue_rearwing_lllb"
-	opacity = FALSE
-
-/obj/structure/dropship_piece/two/rearwing/rightrrbottom
-	icon_state = "blue_rearwing_rrrb"
-	opacity = FALSE
-
-
-/obj/structure/dropship_piece/three
-	name = "\improper Triumph"
-
-/obj/structure/dropship_piece/three/front
-	icon_state = "brown_front"
-	opacity = FALSE
-
-/obj/structure/dropship_piece/three/front/right
-	icon_state = "brown_fr"
-
-/obj/structure/dropship_piece/three/front/left
-	icon_state = "brown_fl"
-
-
-/obj/structure/dropship_piece/three/cockpit/left
-	icon_state = "brown_cockpit_fl"
-
-/obj/structure/dropship_piece/three/cockpit/right
-	icon_state = "brown_cockpit_fr"
-
-
-/obj/structure/dropship_piece/three/weapon
-	opacity = FALSE
-
-/obj/structure/dropship_piece/three/weapon/leftleft
-	icon_state = "brown_weapon_ll"
-
-/obj/structure/dropship_piece/three/weapon/leftright
-	icon_state = "brown_weapon_lr"
-
-/obj/structure/dropship_piece/three/weapon/rightleft
-	icon_state = "brown_weapon_rl"
-
-/obj/structure/dropship_piece/three/weapon/rightright
-	icon_state = "brown_weapon_rr"
-
-
-/obj/structure/dropship_piece/three/wing
-	opacity = FALSE
-
-/obj/structure/dropship_piece/three/wing/left/top
-	icon_state = "brown_wing_lt"
-
-/obj/structure/dropship_piece/three/wing/left/bottom
-	icon_state = "brown_wing_lb"
-
-/obj/structure/dropship_piece/three/wing/right/top
-	icon_state = "brown_wing_rt"
-
-/obj/structure/dropship_piece/three/wing/right/bottom
-	icon_state = "brown_wing_rb"
-
-
-/obj/structure/dropship_piece/three/corner/middleleft
-	icon_state = "brown_middle_lc"
-
-/obj/structure/dropship_piece/three/corner/middleright
-	icon_state = "brown_middle_rc"
-
-/obj/structure/dropship_piece/three/corner/rearleft
-	icon_state = "brown_rear_lc"
-
-/obj/structure/dropship_piece/three/corner/rearright
-	icon_state = "brown_rear_rc"
-
-
-/obj/structure/dropship_piece/three/engine
-	opacity = FALSE
-
-/obj/structure/dropship_piece/three/engine/lefttop
-	icon_state = "brown_engine_lt"
-
-/obj/structure/dropship_piece/three/engine/righttop
-	icon_state = "brown_engine_rt"
-
-/obj/structure/dropship_piece/three/engine/leftbottom
-	icon_state = "brown_engine_lb"
-
-/obj/structure/dropship_piece/three/engine/rightbottom
-	icon_state = "brown_engine_rb"
-
-
-/obj/structure/dropship_piece/three/rearwing/lefttop
-	icon_state = "brown_rearwing_lt"
-
-/obj/structure/dropship_piece/three/rearwing/righttop
-	icon_state = "brown_rearwing_rt"
-
-/obj/structure/dropship_piece/three/rearwing/leftbottom
-	icon_state = "brown_rearwing_lb"
-
-/obj/structure/dropship_piece/three/rearwing/rightbottom
-	icon_state = "brown_rearwing_rb"
-
-/obj/structure/dropship_piece/three/rearwing/leftlbottom
-	icon_state = "brown_rearwing_llb"
-	opacity = FALSE
-
```

Comment:
why is all this stuff being deleted

Reply:
Not deleted, just moved. All this junk was put into it's own file.